api/autopilot.go

```diff
@@ -53,9 +53,12 @@ type (
 
 	// HostsConfig contains all hosts configuration parameters.
 	HostsConfig struct {
-		IgnoreRedundantIPs bool                        `json:"ignoreRedundantIPs"`
-		MaxDowntimeHours   uint64                      `json:"maxDowntimeHours"`
-		ScoreOverrides     map[types.PublicKey]float64 `json:"scoreOverrides"`
+		MaxDowntimeHours uint64                      `json:"maxDowntimeHours"`
+		ScoreOverrides   map[types.PublicKey]float64 `json:"scoreOverrides"`
+
+		// used for testing
+		AcceptZeroDowntimeHours bool `json:"acceptZeroDowntimeHours"`
```

Comment:
Nah we have a check for that, but I'll think of something. I had the same feelings towards it as you have but in the end I figured it was fine. I think mostly because I considered `IgnoreRedundantIPs` more or less the same thing but I can see why it's not.